WKWebViewRTC/Classes/WKWebViewRTC.swift

```diff
@@ -430,7 +427,7 @@ public class WKWebViewRTC : NSObject {
 		self.queue.async { [weak pluginRTCPeerConnection, weak pluginMediaStreamTrack] in
 			pluginRTCPeerConnection?.removeTrack(pluginMediaStreamTrack!)
 			// TODO remove only if not used by other stream
-			self.deleteMediaStreamTrack(trackId)
+			// self.deleteMediaStreamTrack(pluginMediaStreamTrack!)
```

Comment:
@JustDoIt9 Can you give me some context here around the TODO?

Reply:
https://github.com/OpenTelecom/WKWebViewRTC/pull/9#issuecomment-734907041

Reply:
@noahmehl ,  hthetiot mentioned about this.